pytest_blockage.py

```diff
@@ -39,6 +44,34 @@ def whitelisted(self, host, *args, **kwargs):
         httplib.HTTPConnection.__init__ = whitelisted
 
 
+def block_http_aiohttp(whitelist):
+    try:
+        from aiohttp.client import ClientSession
+    except ImportError:
+        # no aiohttp installed
```

Comment:
Add log message at level info here. If one day aiohttp changes import locations, this code will silently fail without giving a hint as to why it fails. better yet, if you want, don't catch for import errors at all but use pkg_resources to check if the aiohttp package is installed.
Then a changed import in aiohttp will trigger an exception which is even better than a log message.